kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java

```diff
@@ -244,6 +245,11 @@ public KubernetesListMixedOperation lists() {
       .withDoneableType(doneClass));
   }
 
+  @Override
+  public RawCustomResourceOperationsImpl rawCustomResource(CustomResourceDefinition customResourceDefinition) {
```

Comment:
I am wondering if this can be named `customResource` instead of `rawCustomResource`?

Reply:
Also it would probably make sense to be able to have the same functionality with just specifying:

- apiGroup
- apiVersion
- kind
- plural  (optionally).